src/test/groovy/com/palantir/gradle/externalpublish/ExternalPublishRootPluginIntegrationSpec.groovy

```diff
@@ -308,6 +342,24 @@ class ExternalPublishRootPluginIntegrationSpec extends IntegrationSpec {
         verifyPomFile(gnv, 'conjure')
     }
 
+    def 'can publish intellij plugin to local maven repo on disk'() {
+        setup:
+        publishIntellij()
+        def mavenRepoDir = testingMavenRepo()
+
+        when:
+        // instrumentCode causes a crash as trying it tries to publish to the intellij marketplace
```

Comment:
I don't think it's actually trying to publish here; there's something horrible happening in the tests probably to do with classloader we do not fully understand with it.